index.md

```diff
@@ -25,7 +25,7 @@ helm repo update
 
 - Run the following command to install the OpenEBS Dynamic LocalPV Provisioner helm chart:
 ```bash
-helm install <your-relase-name> openebs-localpv/localpv-provisioner --namespace openebs --create-namespace
+helm install [RELEASE_NAME] openebs-localpv/localpv-provisioner --namespace [NAMESPACE]
```

Comment:
Which one is correct @niladrih ?

Reply:
The other commands -- uninstall and upgrade follow this pattern.
```
helm install [RELEASE_NAME] openebs-localpv/localpv-provisioner --namespace [NAMESPACE]
```

Reply:
In helm 3 , we have to create the namespace. So don't we need to mention it with steps for namespace creation also. Ref: https://openebs.github.io/cstor-operators/. You can see that namespace needs to be either created earlier or `create-namespace` be used

Reply:
Agreed. Added the flag.

Reply:
Reflected changes in master branch README.md as well.
Ref: https://github.com/openebs/dynamic-localpv-provisioner/pull/46